docs/docs/guides/motion_detection.md

```diff
@@ -0,0 +1,64 @@
+
+:::
+
+### Threshold
+
+The threshold value dictates how much of a change in a pixels luminance is required to be considered motion. 
+
+Lower values mean motion detection is more sensitive to changes in color, making it more likely for exmaple to detect motion when a brown dogs blends in with a brown fence or a person wearing a red shirt blends in with a red car. If the threshold is too low however, it may detect things like grass blowing in the wind, shadows, etc. to be detected as motion.
+
+Watching the motion boxes in the debug view, adjust the threshold until the undesired motion is not detected but be sure to also check with desired motion to ensure it is still detected.
+
+### Contour Area
+
+Once the threshold calculation is run, the pixels that have changed are grouped together. The contour area value is used to decide which groups of changed pixels qualify as motion. Smaller values are more sensitive meaning people that are far away, small animals, etc. are more likely to be detected as motion, but it also means that small changes in shadows, leaves, etc. are detected as motion. Higher values are less sensitive meaning these things won't be detected as motion but with the risk that desired motion won't be detected until closer to the camera.
+
+Watching the motion boxes in the debug view, adjust the contour area until there are no motion boxes smaller than the smallest you'd expect frigate to detect something moving. 
+
+### Improve Contrast
+
+At this point if motion is working as desired there is no reason to continue with tuning for the day. If you were unable to find a balance between desired and undesired motion being detected, you can try disabling improve contrast and going back to the threshold and contour area steps.
+
+## Tuning Motion Detection During The Night
+
+Once daytime motion detection is tuned, there is a chance that the settings will work well for motion detection during the night as well. If this is the case then the preferred settings can be written to the config file and left alone. 
+
+However, if the preferred day settings do not work well at night it is recommended to use HomeAssistant or some other solution to automate changing the settings. That way completely separate sets of motion settings can be used for optimal day and night motion detection.
+
+## Tuning For Large Changes In Motion
+
+Larges changes in motion like PTZ moves and camera switches between Color and IR mode should result in no motion detection. This is done via the `lightning_threshold` configuration. It is defined as the percentage of the image used to detect lightning or other substantial changes where motion detection needs to recalibrate. Increasing this value will make motion detection more likely to consider lightning or ir mode changes as valid motion. Decreasing this value will make motion detection more likely to ignore large amounts of motion such as a person approaching a doorbell camera.
```

Comment:
the second "ir" should be capitalized in this paragraph

Reply:
thanks, just so you know if you select a range of text in the code reviewer you can use the first option on the left to suggest a change, it makes it easier to commit directly

![Screen Shot 2023-06-28 at 13 46 37 PM](https://github.com/blakeblackshear/frigate/assets/14866235/b349b8c2-5575-4ffa-bf48-4e3bb90977ea)

Reply:
Thank you!! I've seen that in other PRs, but couldn't figure out how to do it. I figured it was related to opening the PR in a code space.